src/gwt/org/bbop/apollo/gwt/shared/FeatureStringEnum.java

```diff
@@ -36,7 +37,7 @@
         DATE,
         CURRENT,
         CURRENT_TAB,
-        COMMENT("Comment"),
+        COMMENT("comment"),
```

Comment:
By default the value is lower-case, so you can just get rid of the qualifier altogether.